engines/metaengine.h

```diff
@@ -313,200 +343,226 @@ class MetaEngine : public PluginObject {
+		 * This flag can only be set when kSavesSupportMetaInfo is set.
 		 */
 		kSavesSupportCreationDate,
 
 		/**
-		 * Features 'play_time' entry in the savestate returned by
-		 * querySaveMetaInfo. It indicates how long the user played
-		 * the game till the save.
-		 * This flag may only be set when 'kSavesSupportMetaInfo' is set.
+		 * Feature @c play_time entry in the save state returned by
+		 * querySaveMetaInfo.
+		 *
+		 * This indicates how long the user played the game until the save.
+		 * This flag can only be set when kSavesSupportMetaInfo is set.
 		 */
 		kSavesSupportPlayTime,
 
 		/**
-		* Feature is available if engine's saves could be detected
-		* with "<target>.###" pattern and "###" corresponds to slot
-		* number.
+		* This feature is available if the engine's saves can be detected
+		* with:
+		* @code
+		* <target>.###
+		* @endcode
+		* where ### corresponds to slot number.
 		*
-		* If that's not true or engine is using some unusual way
+		* If that is not true, or if the engine uses some unusual way
 		* of detecting saves and slot numbers, this should be
-		* unavailable. In that case Save/Load dialog for engine's
+		* unavailable. In that case Save/Load dialog for the engine's
 		* games is locked during cloud saves sync.
 		*/
 		kSimpleSavesNames,
 
 		/**
-		 * Uses default implementation of save header and thumbnail
+		 * Use the default implementation of save header and thumbnail
 		 * appended to the save.
+		 *
 		 * This flag requires the following flags to be set:
-		 *   kSavesSupportMetaInfo, kSavesSupportThumbnail, kSavesSupportCreationDate,
-		 *   kSavesSupportPlayTime
+		 * - kSavesSupportMetaInfo
+		 * - kSavesSupportThumbnail
+		 * - kSavesSupportCreationDate
+		 * - kSavesSupportPlayTime
 		 */
 		kSavesUseExtendedFormat
 	};
 
-	//@}
-
 	/**
 	 * Return a list of achievement descriptions for the specified target.
 	 *
 	 * The default implementation returns an empty list.
 	 *
-	 * @param target    name of a config manager target
-	 * @return          a list of achievement descriptions for an engine plugin
-	 *                  and target
+	 * @param target  Name of a config manager target.
+	 *
+	 * @return A list of achievement descriptions for an engine plugin and target.                
 	 */
 	virtual const Common::AchievementsInfo getAchievementsInfo(const Common::String &target) const {
 		return Common::AchievementsInfo();
 	}
 
 	/**
 	 * Determine whether the engine supports the specified MetaEngine feature.
-	 * Used by e.g. the launcher to determine whether to enable the "Load" button.
+	 *
+	 * Used by e.g. the launcher to determine whether to enable the Load button.
 	 */
 	virtual bool hasFeature(MetaEngineFeature f) const;
 
+	/**
+	 * @todo Append extended save (?).
+	 */
 	static void appendExtendedSave(Common::OutSaveFile *saveFile, uint32 playtime, Common::String desc, bool isAutosave);
+	/**
+	 * @todo Parse savegame header to retrieve its description.
```

Comment:
Indeed. It parse the extended savegame header to retrieve the SaveStateDescriptor information (save date, save time, and total play time).